src/datanode/src/sql/alter.rs

```diff
@@ -76,6 +83,9 @@ impl SqlHandler {
             AlterTableOperation::DropColumn { name } => AlterKind::DropColumns {
                 names: vec![name.value.clone()],
             },
+            AlterTableOperation::RenameTable { new_table_name } => AlterKind::RenameTable {
```

Comment:
You could return the InvalidSql error here, then we don't need to add the `RenameTable` variant to `AlterKind` as we only expect to change the parser.